tests/integration/components/aggregate-query-maker-test.js

```diff
@@ -19,36 +19,46 @@ test('it renders', function(assert) {
 
   this.render(hbs`{{aggregate-query-maker startDate="2010-06-01" endDate="2016-07-02" submit=(action mockSubmit) reset=(action mockReset)}}`);
 
-  assert.notEqual(this.$().text(), '');
+  assert.notEqual(this.$().text(), '', 'The component rendered correctly.');
 
 });
 
-/*
-
-test('it submits the query params', function(assert) {
-  const universityVillage = {"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-87.67248630523682,41.86454328565965],[-87.67248630523682,41.872117384500754],[-87.6549768447876,41.872117384500754],[-87.6549768447876,41.86454328565965],[-87.67248630523682,41.86454328565965]]]}};
-  let mockLayer = L.geoJson(universityVillage);
-
-  this.set('mockSubmit', (actual) => {
-    let expected = {
-      geom: JSON.stringify(mockLayer.toGeoJSON()),
-      startDate: '2016-2-10',
-      endDate: '2016-2-10',
-      agg: 'day'
-    };
-    assert.equal(actual.geoJSON, expected.geom, 'Failed to report correct geoJSON');
-    assert.equal(actual.agg, expected.agg, 'Failed to report selected aggregation unit.');
+test('Parameters correctly control the component.', function(assert){
+  this.set('start', '2010-06-10');
+  this.set('end', '2016-07-20');
+  this.set('agg', 'day');
+  this.set('center', 'chicago');
+
+  this.set('mockSubmit', () => {
+    // NoOp
   });
 
   this.set('mockReset', () => {
     // NoOp
   });
 
-  this.set('layer', mockLayer);
+  this.set('cities', {
```

Comment:
I've made a change that eliminates the need to define these empty actions. The problem was that they were being wrapped twice, so the actions (which were already _actions_) being passed to `aggregate-query-maker` were being wrapped in `(action ...)` tags once more when they were passed to `submit-or-reset`, which made them a dependency. By getting rid of the second tag, we only have to wrap the functions with `(action ...)` the first time with the query maker, or not pass any actions at all.

The rest of the settings are the parts being tested, so it seems that they should be there.